Microsoft Edge Add-ons Publish API What's New and Roadmap (#1855)
MicrosoftDocs/edge-developer: microsoft-edge/extensions-chromium/whats-new/released-features.md

For more information about upcoming updates and feature releases, see [Microsoft Edge Add-ons roadmap](roadmap.md).

<!-- ====================================================================== -->

> ## Publish and update extensions to Microsoft Edge Add-ons using APIs

Reviewer: ```suggestion
## Publish and update extensions using the Microsoft Edge Add-ons API
```

> You can integrate APIs directly into your Continuous Integration / Continuous Delivery (CI/CD) pipeline. You can also publish extension package updates to the Microsoft Edge Add-ons website without using Partner Center. The Microsoft Edge Add-ons Publish API is currently in beta.

Reviewer 2: ```suggestion
You can integrate APIs directly into your Continuous Integration / Continuous Delivery (CI/CD) pipeline. You can also publish extension package updates to the Microsoft Edge Add-ons website without using Partner Center.
```

Reviewer: Changed

> 

Reviewer 2: link to the Using API article

Reviewer: added
